packages/api/src/client/createOpenApiRequest.ts

```diff
@@ -55,12 +55,8 @@ export function createOpenApiRequest<
     }
 
     const headersInit: HeadersInit = new Headers();
-    if (requestMediaType != null) {
-      headersInit.set("Content-Type", requestMediaType);
-    }
-    if (responseMediaType != null) {
-      headersInit.set("Accept", responseMediaType);
-    }
+    headersInit.set("Content-Type", requestMediaType ?? "application/json");
+    headersInit.set("Accept", responseMediaType ?? "application/json");
```

Comment:
Something changed in the backend where we weren't able to make requests when the mimetypes were omitted, but only for the loadObjects() call. cc @pashakondratyev